### PR TITLE
WT-5434 Update compatibility_test_for_mongodb_releases.sh for durable history integration into MongoDB

### PR DESCRIPTION
**Pull request by hbokhari**
* Update compatibility_test_for_mongodb_releases.sh for durable history integration into MongoDB

* Remove the deprecated --enable-page-version-ts flag

### CODE REVIEW

**Comment by hbokhari** [comment]
Patch build to test the script changes: https://evergreen.mongodb.com/task/wiredtiger_compatibility_tests_compatibility_test_for_mongodb_releases_patch_97ea5409db823020f4cc90c4b3fc4567d5a84266_5e54949bc9ec445d863bc82d_20_02_25_03_31_11

**Comment by hbokhari** [comment]
@lukech and @quchenhao ,

There are failures in PR testing which are because of changes in this PR. I started special patch build to test the script independent of PR testing.

**Comment by hbokhari** [comment]
I guess we want to finish all tickets that are part of the durable history epic.

**Comment by quchenhao** [comment]
Can we move this ticket out of the epic instead? @bvpvamsikrishna

File: test/evergreen/compatibility_test_for_mongodb_releases.sh [hunk 1]
```diff
@@ -114,15 +108,15 @@ run()
...
 	(build_rel 4.0) || return 1
+	(build_rel 4.2) || return 1
 	(build_rel develop) || return 1
-	(build_rel develop-timestamps) || return 1
 
 	# Verify forward/backward compatibility.
 	(verify 3.4 3.6) || return 1
 	(verify 3.6 4.0) || return 1
+	(verify 4.0 4.2) || return 1
 	(verify 4.0 develop) || return 1
```
Comment (lukech) [inline test/evergreen/compatibility_test_for_mongodb_releases.sh]: I believe we only need to support and verify compatibility between 2 adjacent releases. If `develop` here is equivalent to 4.4, this line should be removed?

Comment (hbokhari) [reply test/evergreen/compatibility_test_for_mongodb_releases.sh]: Just checked with @bvpvamsikrishna and you are correct - we should only be testing 4.4 against 4.2.

**Pull request merged**